homeassistant/components/nexia/__init__.py

```diff
@@ -21,36 +19,14 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-CONFIG_SCHEMA = vol.Schema(
```

Comment:
We should keep `CONFIG_SCHEMA` around, just in case someone still has an entry.

```py
CONFIG_SCHEMA = cv.deprecated(DOMAIN)
```

Reply:
Done 👍